packages/expo-file-system/CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
 ### 🛠 Breaking changes
 
+- `FileSystem.downloadAsync` and `FileSystem.DownloadResumable` now works in the background - they won't reject when the user backgrounds the application. ([#7380](https://github.com/expo/expo/pull/7380) [@lukmccall](https://github.com/lukmccall))
 - `FileSystem.getContentUriAsync` now returns a string. ([#7192](https://github.com/expo/expo/pull/7192) by [@lukmccall](https://github.com/lukmccall))
 
 ### 🎉 New features
 
+- Add `FileSystem.uploadAsync` method. ([#7380](https://github.com/expo/expo/pull/7380) and [#7393](https://github.com/expo/expo/pull/7393) by [@lukmccall](https://github.com/lukmccall))
```

Comment:
```suggestion
- Add `FileSystem.uploadAsync` method. ([#7380](https://github.com/expo/expo/pull/7380) by [@lukmccall](https://github.com/lukmccall))
```

This PR shouldn't reference any other.
If you have some additional PRs that have to merged right after this one you should go this way:
- merge main PR to `master`
- rebase consecutive PR with `master` branch (or even better - always have consequtive PR rebased relatively to the brach of main PR) and merge them (almost) immediately after - this subsequent merge should modify main PR's CHANGELOG entry

Other way you could first merge all subsequent PRs into the main branch and finally merge the main branch to the `master` branch.